src/test/java/wedlog/address/logic/commands/GuestAddCommandIntegrationTest.java

```diff
@@ -26,6 +28,28 @@ public void setUp() {
         model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
     }
 
+    @Test
+    public void execute_newGuestIntoEmptyAddressBook_success() {
+        model = new ModelManager(new AddressBook(), new UserPrefs());
+        Guest validGuest = new GuestBuilder().build();
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.addGuest(validGuest);
+
+        assertCommandSuccess(new GuestAddCommand(validGuest), model,
+                String.format(GuestAddCommand.MESSAGE_SUCCESS, Messages.format(validGuest)),
+                expectedModel);
+
+        Guest insertedGuest = model.getFilteredGuestList().get(0);
+
+        // Check that all fields of the Guest have been stored properly in WedLog
+        assertEquals(insertedGuest.getName(), validGuest.getName());
+        assertEquals(insertedGuest.getPhone(), validGuest.getPhone());
+        assertEquals(insertedGuest.getEmail(), validGuest.getEmail());
+        assertEquals(insertedGuest.getAddress(), validGuest.getAddress());
+        assertEquals(insertedGuest.getTags(), validGuest.getTags());
```

Comment:
Is this line necessary? `assertCommandSuccess` already checks the `model` against the `expectedModel`, which checks between the `ModelManager` objects, which checks between the `AddressBook` objects, and eventually checks between the fields of each `Guest`.
```suggestion
```